code/modules/surgery/bodyparts/helpers.dm

```diff
@@ -89,10 +89,7 @@
 	return TRUE
 
 /mob/living/carbon/human/get_leg_ignore()
-	if(movement_type & FLYING)
-		return TRUE
-	var/obj/item/tank/jetpack/J = get_jetpack()
-	if(J && J.on && !has_gravity())
```

Comment:
Why remove the jetpack stuff?

Reply:
Being in an area without gravity sets the floating var to true, making it redundant.